tools/api-markdown-documenter/src/test/MarkdownDocumenter.test.ts

```diff
@@ -143,18 +142,6 @@ function apiTestSuite(
 					);
 				});
 
-				it("Render Model document (smoke test)", () => {
```

Comment:
These tests were useful early on before we had better test coverage, but they no longer serve much purpose, and would require exporting things that shouldn't be exported.